homeassistant/components/enocean/binary_sensor.py

```diff
@@ -26,31 +36,90 @@
     }
 )
 
+YAML_BINARY_SENSORS_TO_IMPORT = "binary_sensors_to_import"
+
+
+async def async_setup_platform(hass, config, add_entities, discovery_info=None):
+    """Convert the config entry to an entity in the registry."""
+    if not hass.config_entries.async_entries(domain=DOMAIN):
```

Comment:
Here let's just store the dev_id in hass.data and then return. Do all the migration work in `async_setup_entry` instead.

Reply:
I am now storing the config entry instead of a built entity.